### ISSUE TITLE
crash before the launcher starts

### ISSUE THREAD

**Issue by xXxRexRaptorxXx** [aidancbrady/Mekanism#4218]
Here is the console

`[B#349] Running C:\Program Files\Java\jdk1.8.0_45\jre\bin\javaw.exe -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms6144m -Xmx6144m -XX:+UseG1GC -XX:MaxGCPauseMillis=4 -Djava.library.path=E:\Programme\technic\modpacks\botania-1102\bin\natives -Dfml.core.libraries.mirror=http://mirror.technicpack.net/Technic/lib/fml/%s -Dminecraft.applet.TargetDirectory=E:\Programme\technic\modpacks\botania-1102 -Djava.net.preferIPv4Stack=true -cp E:\Programme\technic\modpacks\botania-1102\bin\modpack.jar;E:\Programme\technic\cache\net\minecraft\launchwrapper\1.12\launchwrapper-1.12.jar;E:\Programme\technic\cache\org\ow2\asm\asm-all\5.0.3\asm-all-5.0.3.jar;E:\Programme\technic\cache\jline\jline\2.13\jline-2.13.jar;E:\Programme\technic\cache\com\typesafe\akka\akka-actor_2.11\2.3.3\akka-actor_2.11-2.3.3.jar;E:\Programme\technic\cache\com\typesafe\config\1.2.1\config-1.2.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-actors-migration_2.11\1.1.0\scala-actors-migration_2.11-1.1.0.jar;E:\Programme\technic\cache\org\scala-lang\scala-compiler\2.11.1\scala-compiler-2.11.1.jar;E:\Programme\technic\cache\org\scala-lang\plugins\scala-continuations-library_2.11\1.0.2\scala-continuations-library_2.11-1.0.2.jar;E:\Programme\technic\cache\org\scala-lang\plugins\scala-continuations-plugin_2.11.1\1.0.2\scala-continuations-plugin_2.11.1-1.0.2.jar;E:\Programme\technic\cache\org\scala-lang\scala-library\2.11.1\scala-library-2.11.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-parser-combinators_2.11\1.0.1\scala-parser-combinators_2.11-1.0.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-reflect\2.11.1\scala-reflect-2.11.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-swing_2.11\1.0.1\scala-swing_2.11-1.0.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-xml_2.11\1.0.2\scala-xml_2.11-1.0.2.jar;E:\Programme\technic\cache\lzma\lzma\0.0.1\lzma-0.0.1.jar;E:\Programme\technic\cache\net\sf\jopt-simple\jopt-simple\4.6\jopt-simple-4.6.jar;E:\Programme\technic\cache\java3d\vecmath\1.5.2\vecmath-1.5.2.jar;E:\Programme\technic\cache\net\sf\trove4j\trove4j\3.0.3\trove4j-3.0.3.jar;E:\Programme\technic\cache\com\mojang\netty\1.6\netty-1.6.jar;E:\Programme\technic\cache\oshi-project\oshi-core\1.1\oshi-core-1.1.jar;E:\Programme\technic\cache\net\java\dev\jna\jna\3.4.0\jna-3.4.0.jar;E:\Programme\technic\cache\net\java\dev\jna\platform\3.4.0\platform-3.4.0.jar;E:\Programme\technic\cache\com\ibm\icu\icu4j-core-mojang\51.2\icu4j-core-mojang-51.2.jar;E:\Programme\technic\cache\net\sf\jopt-simple\jopt-simple\4.6\jopt-simple-4.6.jar;E:\Programme\technic\cache\com\paulscode\codecjorbis\20101023\codecjorbis-20101023.jar;E:\Programme\technic\cache\com\paulscode\codecwav\20101023\codecwav-20101023.jar;E:\Programme\technic\cache\com\paulscode\libraryjavasound\20101123\libraryjavasound-20101123.jar;E:\Programme\technic\cache\com\paulscode\librarylwjglopenal\20100824\librarylwjglopenal-20100824.jar;E:\Programme\technic\cache\com\paulscode\soundsystem\20120107\soundsystem-20120107.jar;E:\Programme\technic\cache\io\netty\netty-all\4.0.23.Final\netty-all-4.0.23.Final.jar;E:\Programme\technic\cache\com\google\guava\guava\17.0\guava-17.0.jar;E:\Programme\technic\cache\org\apache\commons\commons-lang3\3.3.2\commons-lang3-3.3.2.jar;E:\Programme\technic\cache\commons-io\commons-io\2.4\commons-io-2.4.jar;E:\Programme\technic\cache\commons-codec\commons-codec\1.9\commons-codec-1.9.jar;E:\Programme\technic\cache\net\java\jinput\jinput\2.0.5\jinput-2.0.5.jar;E:\Programme\technic\cache\net\java\jutils\jutils\1.0.0\jutils-1.0.0.jar;E:\Programme\technic\cache\com\google\code\gson\gson\2.2.4\gson-2.2.4.jar;E:\Programme\technic\cache\com\mojang\authlib\1.5.22\authlib-1.5.22.jar;E:\Programme\technic\cache\com\mojang\realms\1.9.3\realms-1.9.3.jar;E:\Programme\technic\cache\org\apache\commons\commons-compress\1.8.1\commons-compress-1.8.1.jar;E:\Programme\technic\cache\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar;E:\Programme\technic\cache\commons-logging\commons-logging\1.1.3\commons-logging-1.1.3.jar;E:\Programme\technic\cache\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar;E:\Programme\technic\cache\it\unimi\dsi\fastutil\7.0.12_mojang\fastutil-7.0.12_mojang.jar;E:\Programme\technic\cache\org\apache\logging\log4j\log4j-api\2.0-beta9\log4j-api-2.0-beta9.jar;E:\Programme\technic\cache\org\apache\logging\log4j\log4j-core\2.0-beta9\log4j-core-2.0-beta9.jar;E:\Programme\technic\cache\org\lwjgl\lwjgl\lwjgl\2.9.4-nightly-20150209\lwjgl-2.9.4-nightly-20150209.jar;E:\Programme\technic\cache\org\lwjgl\lwjgl\lwjgl_util\2.9.4-nightly-20150209\lwjgl_util-2.9.4-nightly-20150209.jar;E:\Programme\technic\modpacks\botania-1102\bin\minecraft.jar net.minecraft.launchwrapper.Launch --username XxRexRaptorxX --version 1.10.2-forge1.10.2-12.18.3.2185 --gameDir E:\Programme\technic\modpacks\botania-1102 --assetsDir E:\Programme\technic\assets --assetIndex 1.10 --uuid ed5d14eb57c04253a1e1a6dab224b393 --accessToken 62c84dfa8d1041e381d99a0d30b964b0 --userType legacy --tweakClass net.minecraftforge.fml.common.launcher.FMLTweaker --versionType Forge --title Botania 1.10.2 --icon E:\Programme\technic\assets\packs\botania-1102\icon.png
[B#349] C:\Program Files\Java\jdk1.8.0_45\jre\bin\javaw.exe -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms6144m -Xmx6144m -XX:+UseG1GC -XX:MaxGCPauseMillis=4 -Djava.library.path=E:\Programme\technic\modpacks\botania-1102\bin\natives -Dfml.core.libraries.mirror=http://mirror.technicpack.net/Technic/lib/fml/%s -Dminecraft.applet.TargetDirectory=E:\Programme\technic\modpacks\botania-1102 -Djava.net.preferIPv4Stack=true -cp E:\Programme\technic\modpacks\botania-1102\bin\modpack.jar;E:\Programme\technic\cache\net\minecraft\launchwrapper\1.12\launchwrapper-1.12.jar;E:\Programme\technic\cache\org\ow2\asm\asm-all\5.0.3\asm-all-5.0.3.jar;E:\Programme\technic\cache\jline\jline\2.13\jline-2.13.jar;E:\Programme\technic\cache\com\typesafe\akka\akka-actor_2.11\2.3.3\akka-actor_2.11-2.3.3.jar;E:\Programme\technic\cache\com\typesafe\config\1.2.1\config-1.2.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-actors-migration_2.11\1.1.0\scala-actors-migration_2.11-1.1.0.jar;E:\Programme\technic\cache\org\scala-lang\scala-compiler\2.11.1\scala-compiler-2.11.1.jar;E:\Programme\technic\cache\org\scala-lang\plugins\scala-continuations-library_2.11\1.0.2\scala-continuations-library_2.11-1.0.2.jar;E:\Programme\technic\cache\org\scala-lang\plugins\scala-continuations-plugin_2.11.1\1.0.2\scala-continuations-plugin_2.11.1-1.0.2.jar;E:\Programme\technic\cache\org\scala-lang\scala-library\2.11.1\scala-library-2.11.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-parser-combinators_2.11\1.0.1\scala-parser-combinators_2.11-1.0.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-reflect\2.11.1\scala-reflect-2.11.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-swing_2.11\1.0.1\scala-swing_2.11-1.0.1.jar;E:\Programme\technic\cache\org\scala-lang\scala-xml_2.11\1.0.2\scala-xml_2.11-1.0.2.jar;E:\Programme\technic\cache\lzma\lzma\0.0.1\lzma-0.0.1.jar;E:\Programme\technic\cache\net\sf\jopt-simple\jopt-simple\4.6\jopt-simple-4.6.jar;E:\Programme\technic\cache\java3d\vecmath\1.5.2\vecmath-1.5.2.jar;E:\Programme\technic\cache\net\sf\trove4j\trove4j\3.0.3\trove4j-3.0.3.jar;E:\Programme\technic\cache\com\mojang\netty\1.6\netty-1.6.jar;E:\Programme\technic\cache\oshi-project\oshi-core\1.1\oshi-core-1.1.jar;E:\Programme\technic\cache\net\java\dev\jna\jna\3.4.0\jna-3.4.0.jar;E:\Programme\technic\cache\net\java\dev\jna\platform\3.4.0\platform-3.4.0.jar;E:\Programme\technic\cache\com\ibm\icu\icu4j-core-mojang\51.2\icu4j-core-mojang-51.2.jar;E:\Programme\technic\cache\net\sf\jopt-simple\jopt-simple\4.6\jopt-simple-4.6.jar;E:\Programme\technic\cache\com\paulscode\codecjorbis\20101023\codecjorbis-20101023.jar;E:\Programme\technic\cache\com\paulscode\codecwav\20101023\codecwav-20101023.jar;E:\Programme\technic\cache\com\paulscode\libraryjavasound\20101123\libraryjavasound-20101123.jar;E:\Programme\technic\cache\com\paulscode\librarylwjglopenal\20100824\librarylwjglopenal-20100824.jar;E:\Programme\technic\cache\com\paulscode\soundsystem\20120107\soundsystem-20120107.jar;E:\Programme\technic\cache\io\netty\netty-all\4.0.23.Final\netty-all-4.0.23.Final.jar;E:\Programme\technic\cache\com\google\guava\guava\17.0\guava-17.0.jar;E:\Programme\technic\cache\org\apache\commons\commons-lang3\3.3.2\commons-lang3-3.3.2.jar;E:\Programme\technic\cache\commons-io\commons-io\2.4\commons-io-2.4.jar;E:\Programme\technic\cache\commons-codec\commons-codec\1.9\commons-codec-1.9.jar;E:\Programme\technic\cache\net\java\jinput\jinput\2.0.5\jinput-2.0.5.jar;E:\Programme\technic\cache\net\java\jutils\jutils\1.0.0\jutils-1.0.0.jar;E:\Programme\technic\cache\com\google\code\gson\gson\2.2.4\gson-2.2.4.jar;E:\Programme\technic\cache\com\mojang\authlib\1.5.22\authlib-1.5.22.jar;E:\Programme\technic\cache\com\mojang\realms\1.9.3\realms-1.9.3.jar;E:\Programme\technic\cache\org\apache\commons\commons-compress\1.8.1\commons-compress-1.8.1.jar;E:\Programme\technic\cache\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar;E:\Programme\technic\cache\commons-logging\commons-logging\1.1.3\commons-logging-1.1.3.jar;E:\Programme\technic\cache\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar;E:\Programme\technic\cache\it\unimi\dsi\fastutil\7.0.12_mojang\fastutil-7.0.12_mojang.jar;E:\Programme\technic\cache\org\apache\logging\log4j\log4j-api\2.0-beta9\log4j-api-2.0-beta9.jar;E:\Programme\technic\cache\org\apache\logging\log4j\log4j-core\2.0-beta9\log4j-core-2.0-beta9.jar;E:\Programme\technic\cache\org\lwjgl\lwjgl\lwjgl\2.9.4-nightly-20150209\lwjgl-2.9.4-nightly-20150209.jar;E:\Programme\technic\cache\org\lwjgl\lwjgl\lwjgl_util\2.9.4-nightly-20150209\lwjgl_util-2.9.4-nightly-20150209.jar;E:\Programme\technic\modpacks\botania-1102\bin\minecraft.jar net.minecraft.launchwrapper.Launch --username XxRexRaptorxX --version 1.10.2-forge1.10.2-12.18.3.2185 --gameDir E:\Programme\technic\modpacks\botania-1102 --assetsDir E:\Programme\technic\assets --assetIndex 1.10 --uuid ed5d14eb57c04253a1e1a6dab224b393 --accessToken 62c84dfa8d1041e381d99a0d30b964b0 --userType legacy --tweakClass net.minecraftforge.fml.common.launcher.FMLTweaker --versionType Forge --title Botania 1.10.2 --icon E:\Programme\technic\assets\packs\botania-1102\icon.png 
[B#349]  2017-02-19 21:03:47,869 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
[B#349]  2017-02-19 21:03:47,870 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
[B#349]  [21:03:47] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
[B#349]  [21:03:47] [main/INFO] [LaunchWrapper]: Using primary tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
[B#349]  [21:03:47] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLTweaker
[B#349]  [21:03:47] [main/INFO] [FML]: Forge Mod Loader version 12.18.3.2185 for Minecraft 1.10.2 loading
[B#349]  [21:03:47] [main/INFO] [FML]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_45, running on Windows 8.1:amd64:6.3, installed at C:\Program Files\Java\jdk1.8.0_45\jre
[B#349]  [21:03:47] [main/WARN] [FML]: The coremod mekanism.common.launch.LoadingHook does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[B#349] Analytics Response [runModpack]: 200`

**Comment by thommy101** [comment]
Please update your graphics driver, sounds like the NVIDIA bug.

**Comment by black-and-blue** [comment]
Not sure whether this is related, but when I updated my NVIDIA driver in late January, I unable to launch any FTB-based modpack. Did some digging and found this post on the FTB forums:
http://support.feed-the-beast.com/t/can ... pack/29747

The post suggests that rolling back to the previous version of the NVIDIA driver would fix the problem, which I did, and and that fixed the problem.  Unfortunately, I didn't get the logs at the time so I can't confirm whether it's the same issue.

The date of the broken version of the driver: 2017-01-20
The date of the version I rolled back to: 2016-12-11
My specific video card (not sure whether it's specific to this): NVIDIA GeForce GTX 760
